src/annotator/annotation-sync.js

```diff
@@ -18,18 +20,16 @@
  *
  * It also listens for events from Guest when new annotations are created or
  * annotations successfully anchor and relays these to the sidebar app.
+ *
+ * @implements Destroyable
  */
 export default class AnnotationSync {
   /**
-   * @param {Bridge} bridge
-   * @param {Object} options
-   *   @param {(event: string, callback: (...args: any[]) => void) => void} options.on -
-   *     Function that registers a listener for an event from the rest of the
-   *     annotator
-   *   @param {(event: string, ...args: any[]) => void} options.emit -
-   *     Function that publishes an event to the rest of the annotator
+   * @param {EventBus} eventBus - enables intra-frame communication
+   * @param {Bridge} bridge - enables inter-frame communication
```

Comment:
Initially when I read these comments I misread the documentation for `eventBus` and `bridge` as being the same. I didn't notice the `intra-`/`inter-` distinction.

These comments describe what the `EventBus` and `Bridge` classes do generically, but not what they are for in the context of `AnnotationSync`. 

```
eventBus - Event bus for communicating with the annotator code (eg. the Guest)
bridge - Channel for communicating with the sidebar
```